app/src/main/java/com/sunlightlabs/android/congress/utils/Utils.java

```diff
@@ -56,19 +61,20 @@ public static String formatRollId(String id) {
 	}
 	
 	public static Intent legislatorIntent(String id) {
-		return new Intent().setClassName(
+		// eventually go to the legislator pager
+		Intent destination = new Intent().setClassName(
 				"com.sunlightlabs.android.congress",
 				"com.sunlightlabs.android.congress.LegislatorPager")
 			.putExtra("bioguide_id", id);
-	}
 
-	public static Intent legislatorIntent(Context context, Legislator legislator) {
-		return new Intent(context, LegislatorPager.class)
-			.putExtra("bioguide_id", legislator.bioguide_id)
-			.putExtra("legislator", legislator);
+		// but always load the legislator first
+		return new Intent().setClassName(
+				"com.sunlightlabs.android.congress",
+				"com.sunlightlabs.android.congress.LegislatorLoader")
+			.putExtra("id", id)
+			.putExtra("intent", destination);
```

Comment:
This change makes it so legislators are always loaded from network, before the LegislatorPager is created.